Sources/Pokepay/BankAPI/Transaction/CreateTransactionWithCpm.swift

```diff
@@ -4,12 +4,12 @@ public extension BankAPI.Transaction {
     struct CreateWithCpm: BankRequest {
         public let cpmToken: String
         public let accountId: String?
-        public let amount: Double?
+        public let amount: Double
```

Comment:
`CreateWithBill` Request has the same signature as this one.  We should update the amount parameter there also to be consistent with this one.

Reply:
In `CreateWithBill` is a little different., the amount could be null then who transaction with it, that person can input how much he/she want to pay it.